frontends/infinite-corridor/src/pages/FieldPage.tsx

```diff
@@ -94,48 +93,66 @@ const FieldCarousel: React.FC<FieldListProps> = ({ list }) => {
 
 const FieldPage: React.FC = () => {
   const { name } = useParams<RouteParams>()
+  const history = useHistory()
   const { hash } = useLocation()
-
-  const [value, setValue] = React.useState(keyFromHash(hash))
+  const tabValue = keyFromHash(hash)
   const fieldQuery = useFieldDetails(name)
   const handleChange = useCallback(
     (event: React.SyntheticEvent, newValue: string) => {
-      setValue(newValue)
+      history.replace({ hash: newValue })
     },
-    []
+    [history]
   )
 
   const featuredList = fieldQuery.data?.featured_list
   const fieldLists = fieldQuery.data?.lists ?? []
 
   return (
     <FieldPageSkeleton name={name}>
-      <TabContext value={value}>
-        <Container>
-          <Grid container spacing={1}>
-            <Grid item xs={12} sm={9}>
-              <TabList className="page-nav" onChange={handleChange}>
-                <Tab component={Link} to="#" label="Home" value="home" />
-                <Tab component={Link} to="#about" label="About" value="about" />
-              </TabList>
+      <TabContext value={tabValue}>
+        <div className="page-subbanner">
+          <Container>
+            <Grid container spacing={1}>
+              <Grid item xs={12} sm={9}>
+                <TabList className="page-nav" onChange={handleChange}>
+                  <Tab component={Link} to="#" label="Home" value="home" />
+                  <Tab
+                    component={Link}
+                    to="#about"
+                    label="About"
+                    value="about"
+                  />
+                </TabList>
+              </Grid>
             </Grid>
-          </Grid>
-        </Container>
-        <Divider />
+          </Container>
+        </div>
         <Container>
           <Grid container spacing={1}>
             <Grid item xs={12} sm={9}>
-              <TabPanel value="home">
+              <TabPanel value="home" className="page-nav-content">
                 <p>{fieldQuery.data?.public_description}</p>
                 {featuredList && <FieldCarousel list={featuredList} />}
                 {fieldLists.map(list => (
                   <FieldList key={list.id} list={list} />
                 ))}
               </TabPanel>
-              <TabPanel value="about">BBBBBB</TabPanel>
-            </Grid>
-            <Grid item xs={12} sm={3}>
-              Featured Video
+              <TabPanel value="about" className="page-nav-content">
+                At vero eos et accusamus et iusto odio dignissimos ducimus qui
```

Comment:
Remove this?

Reply:
I did put it there deliberately, but probably better to just leave it blank.